src/main/java/io/nats/client/JetStreamReader.java

```diff
@@ -16,7 +16,8 @@
 import java.time.Duration;
 
 /**
- * This interface provides push like ability for a pull consumer. Was the pre-cursor to simplified consume.
+ * This interface provides a simple iterative access to a pull consumer. 
+ * <p>Note: This interface is superseded by  {@link ConsumerContext ConsumerContext}. For examples for <b>recommended usage</b> see {@link JetStream JetStream}.
```

Comment:
This is basically deprecated because of simplifications. Not sure I want anything to call it out.

Reply:
Yes, I want to call it out. Either we dare to say deprecated or euphemistically "superseded".